### ISSUE TITLE
not an issue a question

### ISSUE THREAD

**Issue by mizzo343** [ceelery/Kast#3]
Hey there, I was wondering what the spawn chance is for indoor the facility and what the spawn power is on Kast
Thanks

**Comment by ceelery** [comment]
Of course! There is a max indoor power count of 20 on kast, and the spawn curve looks like this:
![kast_spawncurve](https://github.com/ceelery/Kast/assets/121422867/f1506a04-67ef-4ca0-8829-4f9d9d2c0c14)

**Comment by mizzo343** [comment]
thank you
